migrator/migrations/m_199_to_m_200_clusters_searchable_platform_type_k8s_version/migration_impl.go

```diff
@@ -0,0 +1,79 @@
+package m199tom200
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	newSchema "github.com/stackrox/rox/migrator/migrations/m_199_to_m_200_clusters_searchable_platform_type_k8s_version/schema/new"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/sac"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+var (
+	batchSize = 2000
+	log       = logging.LoggerForModule()
+)
+
+func migrate(database *types.Databases) error {
+	ctx := sac.WithAllAccess(context.Background())
+	db := database.GormDB
+	pgutils.CreateTableFromModel(ctx, db, newSchema.CreateTableClustersStmt)
+
+	return populatePlatformTypeAndK8sVersionColumns(ctx, db)
+}
+
+func populatePlatformTypeAndK8sVersionColumns(ctx context.Context, database *gorm.DB) error {
+	db := database.WithContext(ctx).Table(newSchema.ClustersTableName)
```

Comment:
Ha, we are suppose to read from a database with old schema.

Reply:
I removed the migration because the clusters get updated during a startup process and that process will add values to new columns too.